src/MsrcSecurityUpdates/Public/Get-MsrcVulnerabilityReportHtml.ps1

```diff
@@ -244,7 +244,7 @@ Process {
         #endregion
 
         $cveSectionHtml += $cveSummaryTableHtml -f @(
-            "$($_.CVE)<br><a href=`"http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=$($_.CVE)`">MITRE</a><br><a href=`"https://web.nvd.nist.gov/view/vuln/detail?vulnId=$($_.CVE)`">NVD</a>",
+            "<a href=`"https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/$($_.CVE)`">$($_.CVE)</a><br><a href=`"http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=$($_.CVE)`">MITRE</a><br><a href=`"https://web.nvd.nist.gov/view/vuln/detail?vulnId=$($_.CVE)`">NVD</a>",

```

Comment:
Not breaking but we're using the back-tick in a here-string, that shouldn't be necessary right since quotes don't need to be escaped in a here-string?